content/docs/nav.yml

```
@@ -1,153 +1,154 @@
 - title: التنصيب
   items:
     - id: getting-started
-      title: Getting Started
+      title: البدء مع React
     - id: add-react-to-a-website
-      title: Add React to a Website
+      title: إضافة React للموقع
     - id: create-a-new-react-app
-      title: Create a New React App
+      title: إنشاء تطبيق React جديد
     - id: cdn-links
-      title: CDN Links
+      title: روابط شبكة توزيع المحتوى ( CDN )
 - title: المعايير الرئيسية
   isOrdered: true
   items:
     - id: hello-world
-      title: Hello World
+      title: مثال «أهلًا بالعالم»
     - id: introducing-jsx
-      title: Introducing JSX
+      title: مقدمة إلى JSX
     - id: rendering-elements
-      title: Rendering Elements
+      title: تصيير العناصر
     - id: components-and-props
-      title: Components and Props
+      title: المكونات والخاصيات
     - id: state-and-lifecycle
-      title: State and Lifecycle
+      title: حالة ودورة حياة المكونات
     - id: handling-events
-      title: Handling Events
+      title: معالجة الأحداث
     - id: conditional-rendering
-      title: Conditional Rendering
+      title: التصيير الشرطي (Conditional Rendering)
     - id: lists-and-keys
-      title: Lists and Keys
+      title: القوائم و المفاتيح
     - id: forms
-      title: Forms
+      title: النماذج
     - id: lifting-state-up
-      title: Lifting State Up
+      title: رفع الحالات للمستوى الأعلى
     - id: composition-vs-inheritance
-      title: Composition vs Inheritance
+      title: الفرق بين التركيب والوراثة
     - id: thinking-in-react
-      title: Thinking In React
+      title: التفكير بطريقة React
```

Seems okay for me :smile:, or should we translate `Thinking In React` as:

```suggestion
      title: التفكير على طريقة React
```

or

```suggestion
      title: التفكير بأسلوب React
```

Or as in [Hsoub](https://wiki.hsoub.com/React/thinking_in_react)'s translation:

```suggestion
      title: أسلوب التفكير في React
```